accepted/0000-fallback-kebab-case-command.md

```diff
@@ -0,0 +1,37 @@
+# Fallback to kebab-case command
+
+## Summary
+
+If the command is unknown, then fallback to the kebab-case version of it if it is in the `PATH`.
+
+## Motivation
+
+It allows users to create custom commands for `npm` such as:
+
+```sh
+npm my-command
+```
+
+## Detailed Explanation
+
+Let's say that `npm my-command` is run. Then `npm` should first see if their is a builtin command called `my-command`. If not then it will look through the path for `npm-my-command`. If that fails, then the current fallback will be run:
```

Comment:
what's the advantage of using `npm-my-command`? is this a convention that's used (besides perhaps git)?

Reply:
I didn't completely understand your question. If you were asking why I chose `my-command`, that was just an example. If you meant why custom commands should be a thing: Creating custom `npm` command will allow developers to "extend" `npm` with more CLI features for users.

Reply:
Allowing that would make "npm adds a new top-level command" a breaking change, and would also make `npm`'s interface different on a per-package basis.